Subject: [PATCH] befs, freevxfs: fix breakage introduced by symlink fixes
From: Alexey Dobriyan <adobriyan@gmail.com>

Somehow only prototypes and return values were converted for
befs_follow_link() and vxfs_immed_follow_link().

Signed-off-by: Alexey Dobriyan <adobriyan@gmail.com>
Signed-off-by: Olaf Hering <olh@suse.de>
---

 fs/befs/linuxvfs.c       |    2 +-
 fs/freevxfs/vxfs_immed.c |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

--- linux-vanilla/fs/befs/linuxvfs.c
+++ linux-follow_the_white_rabbit/fs/befs/linuxvfs.c
@@ -461,7 +461,7 @@ befs_destroy_inodecache(void)
  * The data stream become link name. Unless the LONG_SYMLINK
  * flag is set.
  */
-static int
+static void *
 befs_follow_link(struct dentry *dentry, struct nameidata *nd)
 {
 	befs_inode_info *befs_ino = BEFS_I(dentry->d_inode);
--- linux-vanilla/fs/freevxfs/vxfs_immed.c
+++ linux-follow_the_white_rabbit/fs/freevxfs/vxfs_immed.c
@@ -72,7 +72,7 @@ struct address_space_operations vxfs_imm
  * Returns:
  *   Zero on success, else a negative error code.
  */
-static int
+static void *
 vxfs_immed_follow_link(struct dentry *dp, struct nameidata *np)
 {
 	struct vxfs_inode_info		*vip = VXFS_INO(dp->d_inode);

-
To unsubscribe from this list: send the line "unsubscribe linux-kernel" in
the body of a message to majordomo@vger.kernel.org
More majordomo info at  http://vger.kernel.org/majordomo-info.html
Please read the FAQ at  http://www.tux.org/lkml/

